test/integration/targets/supervisorctl/files/sendProcessStdin.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+$1 - <<EOF
```

Comment:
Why is this Python code embedded in a shell script instead of being in a separate .py file?

Reply:
Because it would not be Python 3 ready due to the shebang:
- even if `supervisorctl` Python module isn't Python 3 compatible, this script is Python 3 compatible
- `ansible_python.executable` is the first parameter, it allows to select the right Python executable

Reply:
Since this is just a helper script, is there a reason it needs to use a specific Python interpreter? It would work just fine using `#!/usr/bin/env python`, would it not?

Reply:
It won't when the managed node is a _Python 3 only_ system following [PEP 0394](https://www.python.org/dev/peps/pep-0394/), isn't it ?

Reply:
From that PEP:
> One exception to this is scripts that are deliberately written to be source compatible with both Python 2.x and 3.x. Such scripts may continue to use python on their shebang line without affecting their portability.

Reply:
updated accordingly :)